exploration/open-close-expressions.md

```diff
@@ -195,13 +200,40 @@ Markup is not valid in _declarations_ or _selectors_.
 
 * Introduces two new sigils, the pound sign `#` and the forward slash `/`.
 
```

Comment:
I would not say "elements" (use placeholder instead). HTML has elements, but we don't.

A different way of saying this is:

```suggestion
There is no difference between an "open" and a "standalone" placeholder in terms of syntax.
It is up to the implementation as to whether a given placeholder is treated
as an opening item or an isolated item in the output.
```

Reply:
Updated my suggestions above with `s/elements/placeholders/`.

Reply:
I disagree with this suggestion, at least until we discuss the open questions related to standalone elements. The whole point of this PR is to emphasize that the current proposal is incomplete without our answering these questions.

Note that the "hash and slash" proposal did include a way to express standalone markup, and so did the "plus and minus" proposal. I'm concerned that suddenly we're discussing a different proposal in which standalone markup has been removed.

How about we add the following con instead of the ones you both proposed?

```suggestion
- Doesn't answer how to handle standalone markup; see below.
```

Reply:
@stasm I don't think that's quite correct. The proposal does say how to handle standalone markup, but not in a syntactically distinct way.

I think there is a case to be made for separate standalone syntax, as it allows relatively naive implementations to be created that don't have a list of standalone vs. spanning markup. 

Thus I think your "con" would be better phrased as:

```suggestion
- Standalone placeholders are not syntactically distinct from opening placeholders; 
  implementations require out-of-band information to distinguish standalone 
  from opening placeholders.
```

Reply:
I hope I don't seem contrary but £ is called a pound sign. # is called an octothorpe.
Coming from a country that used to have pounds as a unit of currency.

Reply:
@kenguest In the specification it would be _U+0023 NUMBER SIGN `#`_ because that is its Unicode name. Octothorpe and pound sign are both alternate names 😉

Reply:
> The proposal does say how to handle standalone markup, but not in a syntactically distinct way.

But it’s not *the same* proposal that was called “hash and slash” which was added in #517.

I object how #535 removed the standalone syntax from that proposal, quoting consensus which was not officially discussed.

Reply:
I remain somewhat confused by the desire to remove this documented "con" of the currently proposed design. This PR and our discussions around this issue have made it clear that there's still desire to discuss standalone markup, but the proposed design itself is not changed here. It still does not differentiate in syntax standalone and open placeholders. Why can't we mention that? Either with the original text, or that proposed by @aphillips or myself earlier in this thread.

On a somewhat meta level, in reply to @stasm from https://github.com/unicode-org/message-format-wg/pull/542#discussion_r1413209532:
> The whole point of this PR is to emphasize that the current proposal is incomplete without our answering these questions.

This approach makes this seem like this PR is attempting re-re-define how we do design and how we work with design documents. Thus far, we've been able to iterate on these docs and their proposed solutions while keeping them internally coherent. Witness for instance the changes to this document and its proposed design. Why can't we continue doing so?

We clearly intend to discuss standalone markup further; I don't see how leaving this out helps that. Or are you suggesting that "standalone" and "open" using the same syntax is _not_ a disadvantage of the proposed design?

Reply:
> I remain somewhat confused by the desire to remove this documented "con" of the currently proposed design. This PR and our discussions around this issue have made it clear that there's still desire to discuss standalone markup, but the proposed design itself is not changed here. It still does not differentiate in syntax standalone and open placeholders. 

It used to, however, right? And then #535 changed it, and I’m trying to object how it did it. See my comment in that PR.

I appreciate the attempt to limit the scope of the proposal to just open and close, but I also note that by doing so, we effectively make a decision about standalone.

I’d prefer it if the doc included `{#foo/}`, as it used to previously. But I also acknowledge that we will discuss this further on Monday.

Since I’m away this week, I’ll defer to you and @aphillips. And since I’m typing this on my phone, would you mind taking over so that we’re able to merge this PR soon? Thanks!